src/imitation/scripts/common/reward.py

```diff
@@ -43,6 +53,17 @@ def config_hook(config, command_name, logger):
         res["net_cls"] = default_net
     if "normalize_input_layer" not in config["reward"]["net_kwargs"]:
         res["net_kwargs"] = {"normalize_input_layer": networks.RunningNorm}
+    if "normalize_output_layer" not in config["reward"]["net_kwargs"]:
+        if command_name == "train_preference_comparisons":
+            res["net_kwargs"] = {"normalize_output_layer": networks.RunningNorm}
+        else:
+            res["net_kwargs"] = {"normalize_output_layer": None}
+    else:
+        if command_name != "train_preference_comparisons":
```

Comment:
Pleasingly I think the idea I had of overriding `predict` rather than normalizing at the `forward()` level would make this work with GAIL too, in which case I think we can enable it by default everywhere and remove this check.

Reply:
Thanks for pointing that out. I recently find out that you are right.